scalr/data_source_scalr_webhook_test.go

```diff
@@ -45,8 +45,8 @@ func TestAccWebhookDataSource_basic(t *testing.T) {
 						"data.scalr_webhook.test-new", "enabled", "false"),
 					resource.TestCheckResourceAttr(
 						"data.scalr_webhook.test-new", "url", "https://example.com/webhook"),
-					resource.TestCheckResourceAttr(
-						"data.scalr_webhook.test-new", "secret_key", ""),
+					resource.TestCheckResourceAttrSet(
+						"data.scalr_webhook.test-new", "secret_key"),
```

Comment:
? https://github.com/Scalr/terraform-provider-scalr/pull/358/files

Reply:
It's a fix for previous PR. secret_keys is not empty in the state.